store/tvShows.ts

```diff
@@ -15,17 +21,20 @@ export const state = (): TvShowsState => ({
   seasonEpisodes: []
 });
 
-type MutationPayload = TvShowsState;
+type MutationPayload = {
```

Comment:
Why?

Reply:
On L95-97 when we commit `ADD_TVSHOW_SEASON_EPISODES` we pass an array of episodes. Previously, with the old `MutationPayload` we ended up with `state.seasonEpisodes` being an array of episodes, instead of an array of array of episodes which is what is expected by `SeasonTabs.vue`.

Reply:
Why would we want array of array of episodes?

Reply:
`SeasonTabs.vue` expects something in the following format:

![image](https://user-images.githubusercontent.com/18101008/102061751-c2382380-3deb-11eb-9633-909ec3a468c8.png)

Then we can iterate through each array to get the relevant episodes for each season, and display them in the relevant tab